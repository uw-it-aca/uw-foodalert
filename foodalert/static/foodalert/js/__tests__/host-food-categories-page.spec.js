"use strict";
import Vue from 'vue';
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue);

import {mount} from '@vue/test-utils';
import FoodCategoriesPage from '../pages/host/food-categories-page.vue';

describe('Food Categories Page', function () {
        test('renders correctly', function () {
            var wrapper = mount(FoodCategoriesPage, {
            });
            expect(wrapper.element).toMatchSnapshot();
        });
})