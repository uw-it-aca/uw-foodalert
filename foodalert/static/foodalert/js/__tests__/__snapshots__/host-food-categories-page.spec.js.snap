// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Food Categories Page renders correctly 1`] = `
<div
  class="page"
>
  <div
    id="standard-notification"
  >
    <div
      class="collapse"
      id="notif-container"
      style="display: none;"
    >
      <div
        class="text-center py-3 container-fluid"
      >
        <div
          class="container"
        >
          <div
            class="row justify-content-center"
          >
            <div
              class="col-md-8"
            >
              <div
                class="h5 text-white pt-2"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
   
  <div
    class="page-content pb-2"
  >
     
    <div
      class="container"
    >
      <div
        class="row justify-content-center"
      >
        <div
          class="page-content-padding col-md-8"
        >
          <h1
            class="pt-4 pb-2"
            id="standard-heading"
          >
            
        Tell us more about your food
    
          </h1>
           
          <div
            id="standard-body"
          >
            <p>
              
            The following will help determine whether your food can be shared with the public.
        
            </p>
             
            <fieldset
              class="form-group"
            >
              <!---->
              <div
                role="group"
                tabindex="-1"
              >
                <div
                  class="custom-control custom-checkbox"
                >
                  <input
                    autocomplete="off"
                    class="custom-control-input"
                    type="checkbox"
                    value="non-perishable"
                  />
                  <label
                    class="custom-control-label"
                  >
                    <span>
                      
                    My food is non-perishable.
                    
                      <a
                        aria-controls="non-perishable"
                        aria-expanded="false"
                        class=""
                        herf="#"
                        href="#"
                        role="button"
                        target="_self"
                      >
                         Examples
                      </a>
                    </span>
                  </label>
                </div>
                 
                <div
                  class="custom-collapse collapse-text-box collapse"
                  id="non-perishable"
                  style="display: none;"
                >
                  <div
                    class="custom-text-box"
                  >
                    
            Candy, beverages (pasteurized, canned, or bottled), chips, dips.
            
                  </div>
                </div>
                 
                <div
                  class="mt-3 custom-control custom-checkbox"
                >
                  <input
                    autocomplete="off"
                    class="custom-control-input"
                    type="checkbox"
                    value="pre-packaged"
                  />
                  <label
                    class="custom-control-label"
                  >
                    <span>
                      
                    My food was commercially pre-packaged.
                    
                      <a
                        aria-controls="pre-packaged"
                        aria-expanded="false"
                        class=""
                        herf="#"
                        href="#"
                        role="button"
                        target="_self"
                      >
                         Examples
                      </a>
                    </span>
                  </label>
                </div>
                 
                <div
                  class="custom-collapse collapse-text-box collapse"
                  id="pre-packaged"
                  style="display: none;"
                >
                  <div
                    class="custom-text-box"
                  >
                    
                Wrapped or boxed baked goods (cakes, pies), chips, store-bought ice cream.
            
                  </div>
                </div>
                 
                <div
                  class="mt-3 custom-control custom-checkbox"
                >
                  <input
                    autocomplete="off"
                    class="custom-control-input"
                    type="checkbox"
                    value="at-home"
                  />
                  <label
                    class="custom-control-label"
                  >
                    
                My food was prepared at home.
            
                  </label>
                </div>
                 
                <div
                  class="mt-3 custom-control custom-checkbox"
                >
                  <input
                    autocomplete="off"
                    class="custom-control-input"
                    type="checkbox"
                    value="none"
                  />
                  <label
                    class="custom-control-label"
                  >
                    <span>
                      
                    None of the above.
                
                    </span>
                  </label>
                </div>
                <!---->
                <!---->
                <!---->
              </div>
            </fieldset>
          </div>
           
          <div
            style="margin-top: 32px;"
          >
            <div
              class="row justify-content-between"
            >
              <div
                class="col-md-4 col-lg-3 order-md-2"
              >
                <button
                  class="btn mb-3 btn-primary btn-lg btn-block disabled"
                  disabled="disabled"
                  type="submit"
                >
                  Continue
                </button>
              </div>
               
              <div
                class="col-md-4 col-lg-3 order-md-1"
              >
                <button
                  class="btn hh-back-button btn-outline-secondary btn-lg btn-block"
                  type="submit"
                >
                  Back
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
   
  <footer
    class="text-center"
    id="relative-footer"
  >
    <a
      href="mailto:help@uw.edu?subject=Hungry Husky support"
    >
      Contact support
    </a>
     
    <p>
      Â© 2019 University of Washington
    </p>
  </footer>
</div>
`;
